py-polars/docs/source/reference/datatypes.rst

```diff
@@ -47,6 +49,17 @@ Nested
     List
     Struct
 
+String
+~~~~~~
+.. autosummary::
+    :toctree: api/
+    :nosignatures:
+
+    String
+    Categorical
+    Enum
```

Comment:
I wouldn't put these under String. They could be in their own section Categorical if you like.

Reply:
If you look at the docs output it's more clear why it's reasonable to group them together; they both _explicitly_ describe themselves as representing strings:

* [Categorical](https://docs.pola.rs/py-polars/html/reference/api/polars.Categorical.html#polars.Categorical): A categorical encoding of a set of strings.
* [Enum](https://docs.pola.rs/py-polars/html/reference/api/polars.Enum.html#polars.Enum): A fixed set categorical encoding of a set of strings.

Reply:
Hm, I guess it's fine then. In my mind the fact that they only work on string data is an implementation detail that may change in the future. But we can update the docs when that happens.